README.md

```diff
@@ -4,7 +4,7 @@
 <sup>-</sup>
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
-# Refined Bitbucket
+![Refined BitBucket](./logo-full.svg)

```

Comment:
GitHub won't be able to recognize this relative path, it has to be a URL pointing to the resource, use https://rawgit.com/ to get a production URL by giving it the location of the raw file of your latest commit, for example: https://github.com/clarkd/refined-bitbucket/blob/a5c4a2e9859f4e6048308d3ab8ee0ef2ad805c1b/logo-full.svg.

We would end up with something like this:

```markdown
![Refined BitBucket](https://cdn.rawgit.com/clarkd/refined-bitbucket/a5c4a2e9859f4e6048308d3ab8ee0ef2ad805c1b/logo-full.svg)
```

Reply:
Notably, they do support relative URLs like this - take a look here for example https://github.com/clarkd/refined-bitbucket/tree/work/dc/logos

Reply:
**My bad, you are correct**. Maybe it was fixed recently, but that was definitely an issue at some point I think (unless I was asleep when I first stumbled upon it).

However it still persists though in the pull request preview, which led to my confusion:

![image](https://user-images.githubusercontent.com/7514993/37151711-183f571a-22ad-11e8-8280-d0ce73fde6e3.png)

Reply:
No worries - took me a few tries to get it working and lots of differing opinions on the best way to do it online!